transformation/src/test/test-files/outputs/antibody/part-1.json

```diff
@@ -1,11 +1,6 @@
-{"award":"/awards/U54HG006998/","clonality":"monoclonal","cross_references":["AR:AB_262044"],"host_organism":"/organisms/mouse/","id":"ENCAB697XQW","isotype":"IgG1","lab":"/labs/richard-myers/","lot_id":"SLBK1346V","product_id":"F1804","purification_methods":["affinity"],"source":"/sources/sigma/","time_created":"2015-12-16T18:10:57.886604Z"}
-{"award":"/awards/UM1HG009390/","clonality":"monoclonal","cross_references":[],"host_organism":"/organisms/rabbit/","id":"ENCAB140BWE","lab":"/labs/bradley-bernstein/","lot_id":"9","product_id":"9751BC","purification_methods":[],"source":"/sources/cell-signaling/","target":"H3K4me3","time_created":"2017-12-06T21:08:41.675624Z"}
+{"award":"/awards/ROADMAP/","clonality":"polyclonal","cross_references":["AR:AB_2118291"],"host_organism":"/organisms/rabbit/","id":"ENCAB000APE","isotype":"IgG","lab":"/labs/roadmap-epigenomics/","lot_id":"509313","product_id":"ab4729","purification_methods":["affinity"],"source":"/sources/abcam/","targets":["H3K27ac"],"time_created":"2013-09-13T13:23:28.986667-08:00"}
+{"award":"/awards/ROADMAP/","clonality":"polyclonal","cross_references":["AR:AB_306848"],"host_organism":"/organisms/rabbit/","id":"ENCAB000APY","isotype":"IgG","lab":"/labs/roadmap-epigenomics/","lot_id":"877391","product_id":"ab8898","purification_methods":["affinity"],"source":"/sources/abcam/","targets":[],"time_created":"2013-09-13T13:23:31.478561-08:00"}
+{"award":"/awards/ROADMAP/","clonality":"polyclonal","cross_references":["AR:AB_310624"],"host_organism":"/organisms/rabbit/","id":"ENCAB000ASB","isotype":"IgG","lab":"/labs/roadmap-epigenomics/","lot_id":"DAM1514011","product_id":"07-449","purification_methods":["Protein A"],"source":"/sources/millipore/","targets":["H3K27me3"],"time_created":"2013-09-13T13:23:22.96603-08:00"}
+{"award":"/awards/U54HG004570/","clonality":"polyclonal","cross_references":["AR:AB_306966","UCSC-ENCODE-cv:H3K36me3"],"host_organism":"/organisms/rabbit/","id":"ENCAB000ADU","isotype":"IgG","lab":"/labs/bradley-bernstein/","lot_id":"499302","product_id":"ab9050","purification_methods":["affinity"],"source":"/sources/abcam/","targets":["H3K36me3"],"time_created":"2014-02-06T16:45:21.384879Z"}
+{"award":"/awards/U54HG006991/","clonality":"polyclonal","cross_references":["AR:AB_1977252"],"host_organism":"/organisms/rabbit/","id":"ENCAB000BLF","lab":"/labs/bradley-bernstein/","lot_id":"DAM1644057","product_id":"07-473","purification_methods":[],"source":"/sources/millipore/","targets":["H3K4me3"],"time_created":"2013-09-13T16:21:43.5051-08:00"}
+{"award":"/awards/U54HG006991/","clonality":"polyclonal","cross_references":["AR:AB_306847"],"host_organism":"/organisms/rabbit/","id":"ENCAB000APS","isotype":"IgG","lab":"/labs/bradley-bernstein/","lot_id":"659352","product_id":"ab8895","purification_methods":["affinity"],"source":"/sources/abcam/","targets":["H3K4me1"],"time_created":"2014-02-05T07:20:12.440209Z"}
```

Comment:
Is the difference in line count expected here?

Reply:
I deleted the `inputs/AntibodyLot/part1.json` file that probably got left behind when the new `inputs/AntibodyLot/part-1.json` file was auto-generated. It looked like some of the values in this file had been auto-generated based on that old input file, so I removed them when I deleted the old input file.